src/frontends/pytorch/src/op/dequantize.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+#include "openvino/op/add.hpp"
+#include "openvino/op/clamp.hpp"
+#include "openvino/op/convert.hpp"
+#include "openvino/op/convert_like.hpp"
+#include "openvino/op/divide.hpp"
+#include "openvino/op/multiply.hpp"
+#include "openvino/op/round.hpp"
+#include "openvino/op/subtract.hpp"
+#include "utils_quantize.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+namespace op {
+
+using namespace ov::op;
+
+OutputVector translate_dequantize(const NodeContext& context) {
+    num_inputs_check(context, 1, 1);
+    const auto input = context.get_input(0);
+
+    std::shared_ptr<ov::Node> output;
+    std::shared_ptr<QuantizedPtNode> quantized_pt_node;
+    if ((quantized_pt_node =
+             cast_quantized_fw_node(input.get_node_shared_ptr(), QuantizedPtNode::quantize_per_tensor))) {
+        const auto input_convert_f32 = context.mark_node(
```

Comment:
Since non-quantized ops can be in-between of quantized ops we need to have DFS lookup to find `QuantizedPtNode` instance which hold parameters of this quantized tensor.

Reply:
Done!